WooCommerce/Classes/ServiceLocator/PushNotification.swift

```diff
@@ -65,3 +101,43 @@ extension PushNotification: Identifiable {
         noteID
     }
 }
+
+#if DEBUG
+extension PushNotification {
+    static func scheduleTestingOrderNotification() {
```

Comment:
Would this be removed or are you planning to leave it for future testing/debug?

Reply:
It could be used for future testing so I have a dilemma if I should remove it or not.

Reply:
I rather test things from real notifications to make sure that everything is working as expected! I would vote to remove it!